igvm/host.py

```diff
@@ -154,7 +155,7 @@ def acquire_lock(self, allow_fail=False):
             )
 
     def release_lock(self):
-        self.dataset_obj['igvm_locked'] = False
+        self.dataset_obj['igvm_locked'] = None
```

Comment:
Is this the same as delattr()?

Reply:
This is okay. Not set attributes are None so setting it to None is the correct way.

`>>> from adminapi.dataset import Query
>>> print(Query().new_object(servertype='vm'))
DatasetObject({'responsible_admin': MultiAttr(), 'no_monitoring': False, 'project_network': None, 'puppet_disabled': False, 'datacenter': None, 'floating_addresses': MultiAttr(), 'nagios_groups': MultiAttr(), 'servertype': 'vm', 'ssh_environment': None, 'monitoring_queue': None, 'iggop_locked': False, 'project_domain': None, 'disk_size_gib': 6, 'nagios_admins': MultiAttr({'admins'}), 'repositories': MultiAttr({'int:innogames:stable'}), 'backup_prio': 300, 'puppet_classes': MultiAttr(), 'project': None, 'acl': MultiAttr(), 'game_type': None, 'auto_update': True, 'allow_to': MultiAttr(), 'served_markets': MultiAttr(), 'backup_storage': None, 'intern_ip': None, 'provider_network': None, 'backup_canshrink': False, 'os': 'stretch', 'backup_window': MultiAttr({'2-5'}), 'monitoring_queues': MultiAttr(), 'backup_sum': 3, 'monitoring_admins': MultiAttr(), 'bladecenter_slot': MultiAttr(), 'mac': MultiAttr(), 'instance': None, 'object_id': None, 'ipv4_only': False, 'legal_personal_data': None, 'network_zones': MultiAttr(), 'team': None, 'ssh_groups': MultiAttr(), 'num_cpu': 4, 'memory': 2048, 'description': None, 'puppet_master': 'master-puppet.ig.local', 'security_confidentiality': None, 'backup_disabled': False, 'hypervisor_standby': None, 'loadbalancer': MultiAttr(), 'environment': 'production', 'monitoring_checks': MultiAttr({'ping'}), 'os_product_license': None, 'domain': MultiAttr(), 'state': 'online', 'puppet_environment': None, 'graphite_graphs': MultiAttr(), 'function': None, 'security_availability': None, 'hypervisor': None, 'puppet_ca': 'ca.puppet.ig.local', 'hostname': None, 'allow_from': MultiAttr(), 'barman_host': None, 'network_zone': None, 'served_game': None, 'game_world': None, 'route_network': None, 'security_integrity': None, 'vlan_interfaces': MultiAttr(), 'ssh_users': MultiAttr(), 'primary_ip6': None, 'backup_freq': MultiAttr(), 'sshfp': MultiAttr(), 'backup_maxage': 36, 'game_market': None, 'subproject': None, 'fixup_enabled': False, 'igvm_locked': False, 'bladecenter': None})
`

Reply:
I think so. I'd use None too.

Reply:
Probably best to also test this against testing serveradmin with a datetime attribute though.

Reply:
I tested it, this is fine.